src/core/lib/iomgr/tcp_server_posix.cc

```diff
@@ -221,7 +224,13 @@ static void on_read(void* arg, grpc_error_handle err) {
     }
 
     if (sp->server->memory_quota->IsMemoryPressureHigh()) {
-      gpr_log(GPR_INFO, "Drop incoming connection: high memory pressure");
+      int64_t dropped_connections_count = ++num_dropped_connections;
```

Comment:
Don't we have to load the value from the atomic? Please also add Craig to review atomic usage.

Reply:
Sure I can add craig. I don't think it matters because the ++ operator is equivalent to fetch_add and we can probably tolerate race conditions also. I made it explicit now anyway